### PR TITLE
Don't activate PIV Card before biometric comparison

### LINKED ISSUE #389: "one-to-one authentication SHALL occur before a PIV Card is issued and activated
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 835-838
 
**Comment (Include rationale for comment)**: "Before the PIV Card is provided to the applicant, the issuer SHALL perform a one-to-one comparison of the applicant against biometric data records available on the PIV Card or in the PIV enrollment record."

As the language stands, it would seem that it is okay to perform a one-to-one biometric authentication after the card is printed and activated, but before an issuer hands the card (or ships the card) to an individual. 

**Suggested Change**: Recommend that the language be updated to explicitly require that a one-to-one authentication SHALL occur before a PIV Card is issued and activated, to ensure it is the same identity being issued to, before any changes occur to the card or identity record. 
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### PR DESCRIPTION
**Pull request by gfiumara**
Would close #389, but requires discussion. Is this possible?

I assume that during the activation process, the facial image, iris images, and fingerprint templates are sitting as files somewhere, unsigned. This PR assumes that we can perform a comparison against them, and if positive, sign them and write them to the card. This also assumes that "activated" means signing things on the card.

### CODE REVIEW

File: _FIPS201/requirements.md [hunk 1]
```diff
@@ -331,11 +331,11 @@ of the federal department or agency.
 + Biometric data used to personalize the PIV Card **SHALL** be those captured during the identity proofing and registration process.
 + During the issuance process, the issuer **SHALL** verify that the individual to whom the PIV Card is to be
     issued is the same as the intended applicant/recipient as approved by the appropriate authority.
-    Before the PIV Card is provided to the applicant, the issuer **SHALL** perform a one-to-one comparison of the
-    applicant against biometric data records available on the PIV Card or in the PIV enrollment record. The one-to-one
+    Before the PIV Card is activated and provided to the applicant, the issuer **SHALL** perform a one-to-one comparison of the
+    applicant against biometric data records in the PIV enrollment record or unsigned biometric data records that would be transferred to and signed on the PIV Card. The one-to-one
```
Comment (gfiumara) [inline _FIPS201/requirements.md:335]:
```suggestion
    applicant against biometric data records in the PIV enrollment record. The one-to-one
```

File: _FIPS201/requirements.md [hunk 2]
```diff
@@ -331,11 +331,11 @@ of the federal department or agency.
...
     comparison requires either a comparison of fingerprints or, if unavailable, other optional biometric data records that are
     available. Minimum accuracy requirements for the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
-    a positive biometric verification decision, the PIV Card **SHALL** be released to the applicant. If the biometric verification decision is negative, or if
+    a positive biometric verification decision, the biometric data records **SHALL** be signed and the PIV Card **SHALL** be activated and released to the applicant. If the biometric verification decision is negative, or if
```
Comment (gfiumara) [inline _FIPS201/requirements.md:338]:
```suggestion
    a positive biometric verification decision the PIV Card **SHALL** be activated and released to the applicant. If the biometric verification decision is negative, or if
```

File: _FIPS201/requirements.md [hunk 3]
```diff
@@ -331,11 +331,11 @@ of the federal department or agency.
...
     issued is the same as the intended applicant/recipient as approved by the appropriate authority.
-    Before the PIV Card is provided to the applicant, the issuer **SHALL** perform a one-to-one comparison of the
-    applicant against biometric data records available on the PIV Card or in the PIV enrollment record. The one-to-one
+    Before the PIV Card is activated and provided to the applicant, the issuer **SHALL** perform a one-to-one comparison of the
+    applicant against biometric data records in the PIV enrollment record. The one-to-one
     comparison requires either a comparison of fingerprints or, if unavailable, other optional biometric data records that are
     available. Minimum accuracy requirements for the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On
-    a positive biometric verification decision, the PIV Card **SHALL** be released to the applicant. If the biometric verification decision is negative, or if
+    a positive biometric verification decision the PIV Card **SHALL** be activated and released to the applicant. If the biometric verification decision is negative, or if
```
Comment (gfiumara) [inline _FIPS201/requirements.md:338]:
```suggestion
    a positive biometric verification decision, the PIV Card **SHALL** be activated and released to the applicant. If the biometric verification decision is negative, or if
```

**Comment by gfiumara** [comment]
We are happy with this change, but @hferraio will check with a card manufacturer to see if this text is valid before merging.

**Comment by gfiumara** [comment]
Updated notes for discussion:
 * To perform biometric comparisons on an non-activated card, the biometric data object would need to be set to PIN and admin access. It is currently PIN only. This is a backwards compatible change.
 * Implementations that perform activation before biometric comparison would be invalidated by this change.
 * Comparing biometrics to the enrollment record does not _guarantee_ that the prints that made their way to the card are from the same person.
 * In some situations, fingerprints may not be in the enrollment record.

Can we impose some sort of requirement like:

 > The activated PIV Card **SHALL** not be placed within arms reach of the applicant until after a positive biometric comparison of the applicant to the biometric data stored on the PIV Card.

This may be an alternate way to resolve the issue.

**Comment by gfiumara** [comment]
We learned that PIV Card personalization include only printing on the outside of the card. Biometric templates and the face/iris images are not placed on the card until activation, necessitating 4efb6fa.

**Comment by jglosx3** [comment]
Issue declined

**Pull request closed without merging**